lib/open-amp/CMakeLists.txt

```diff
@@ -5,4 +5,11 @@
 #
 
 zephyr_include_directories_ifdef(CONFIG_OPENAMP_RSC_TABLE .)
-zephyr_sources_ifdef(CONFIG_OPENAMP_RSC_TABLE resource_table.c)
+# include vendor-specific resource table files
+if(CONFIG_OPENAMP_VENDOR_RSC_TABLE_ENTRY)
+  zephyr_include_directories(vendor_rsc_table)
+  zephyr_sources(vendor_rsc_table/${CONFIG_OPENAMP_VENDOR_RSC_TABLE_FILE})
+elseif(CONFIG_OPENAMP_RSC_TABLE)
+# include generic resource table
+  zephyr_sources(resource_table.c)
+endif()
```

Comment:
seems that `CONFIG_OPENAMP_VENDOR_RSC_TABLE_ENTRY` is now uselless, or i missed something? 

```
if(CONFIG_OPENAMP_RSC_TABLE)
if(CONFIG_OPENAMP_VENDOR_RSC_TABLE_FILE)
  zephyr_include_directories(vendor_rsc_table)
  zephyr_sources(vendor_rsc_table/${CONFIG_OPENAMP_VENDOR_RSC_TABLE_FILE})
else()
  # include generic resource table
  zephyr_sources(resource_table.c)
endif()
endif()
```

Reply:
I've updated the code and now `CONFIG_OPENAMP_VENDOR_RSC_TABLE` depens on `CONFIG_OPENAMP_RSC_TABLE`.